x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/prebuilt_rules/prebuilt_rules_preview.cy.ts

```diff
@@ -1049,5 +1062,104 @@ describe('Detection rules, Prebuilt Rules Installation and Update workflow', ()
         }
       );
     });
+
+    describe('Viewing rule changes in JSON diff view', { tags: TEST_ENV_TAGS }, () => {
+      it('User can see changes in a side-by-side JSON diff view', () => {
+        clickRuleUpdatesTab();
+
+        openRuleUpdatePreview(OUTDATED_RULE_1['security-rule'].name);
+        assertSelectedPreviewTab('JSON view');
+
+        cy.get(UPDATE_PREBUILT_RULE_PREVIEW)
+          .contains('Current rule')
+          .as('currentRuleColumnHeading');
+        cy.get(UPDATE_PREBUILT_RULE_PREVIEW)
+          .contains('Elastic update')
+          .as('elasticUpdateColumnHeading');
+
+        cy.get('@currentRuleColumnHeading').should('be.visible');
+        cy.get('@elasticUpdateColumnHeading').should('be.visible');
+
+        /* "Current rule" should be the left column, "Elastic update" should be the right column. */
+        cy.get('@currentRuleColumnHeading').then(($currentRuleColumnHeading) => {
+          const currentRuleLeftOffset = $currentRuleColumnHeading.offset()?.left || 0;
+          cy.get('@elasticUpdateColumnHeading').then(($elasticUpdateColumnHeading) => {
+            const elasticUpdateLeftOffset = $elasticUpdateColumnHeading.offset()?.left || 0;
+            expect(currentRuleLeftOffset).to.be.lessThan(elasticUpdateLeftOffset);
+          });
+        });
+
+        cy.get(UPDATE_PREBUILT_RULE_PREVIEW).contains('"version": 1').as('currentRuleValue');
+        cy.get(UPDATE_PREBUILT_RULE_PREVIEW).contains('"version": 2').as('updateVersionValue');
+
+        cy.get('@currentRuleValue').should('be.visible');
+        cy.get('@updateVersionValue').should('be.visible');
+
+        /*
+          "Current rule" version value should be displayed in the left column, 
+          "Elastic update" version value should be displayed the right column.
+        */
+        cy.get('@currentRuleValue').then(($currentRuleValue) => {
+          const currentRuleLeftOffset = $currentRuleValue.offset()?.left || 0;
+          cy.get('@updateVersionValue').then(($updateVersionValue) => {
+            const updateVersionLeftOffset = $updateVersionValue.offset()?.left || 0;
+            expect(currentRuleLeftOffset).to.be.lessThan(updateVersionLeftOffset);
+          });
+        });
```

Comment:
Same here, but for the `Diff` library. I think this is a level of detail greater than we should have for e2e tests as smoke tests.

Reply:
Agreed. Removed.